src/stylesheets/core/_functions.scss

```diff
@@ -731,6 +736,7 @@ colors: project and system
 */
 
 @function color($props...){
+  $props: unpack($props);
```

Comment:
FYI: `unpack()` converts nested arglists into single one-dimensional lists for proper parsing:

((A, B), C) --> (A, B, C)